### ISSUE TITLE
feat: add a search for advancements

### ISSUE THREAD

**Issue by Exploity** [42atomys/fabric-advancementinfo-reloaded#8]
how it was done in the original AdvancementInfo mod
![kuva](https://github.com/user-attachments/assets/84dae2a2-a393-4de1-927d-9154596f41ee))

**Comment by 42atomys** [comment]
Hi @Exploity thanks for your feature request, that makes sense and I totally forgot to add it in the readme as "future implémentation" 

My idea are to provide a search system more powerful than the original mod, I let the issue open to collect feedback

**Comment by ItzSkyReed** [comment]
yes please 🙏, this is the only function that is missing and because of which I have to use the original advancement info

**Comment by 42atomys** [comment]
What a good news, the search feature are coming in the next update `0.9.0` 🎉
